addons/mail/static/src/services/new_message_service/new_message_service.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+export const newMessageService = {
```

Comment:
Do we really need a new service for this actually? This should be part of ChatWindowService maybe while legacy, or even messaging service directly.

I don't see any usage of newEnv in the code, so having it in a legacy service would be ok?

Reply:
we need access to the "command" service which is in the new env.